SimPEG/potential_fields/gravity/_numba_functions.py

```diff
@@ -0,0 +1,256 @@
+        )
+
+    Parameters
+    ----------
+    receivers : (n_receivers, 3) array
+        Array with the locations of the receivers
+    nodes : (n_active_nodes, 3) array
+        Array with the location of the mesh nodes.
+    densities : (n_active_cells)
+        Array with densities of each active cell in the mesh.
+    fields : (n_receivers) array
+        Array full of zeros where the gravity fields on each receiver will be
+        stored. This could be a preallocated array or a slice of it.
+    cell_nodes : (n_active_cells, 8) array
+        Array of integers, where each row contains the indices of the nodes for
+        each active cell in the mesh.
+    kernel_func : callable
+        Kernel function that will be evaluated on each node of the mesh. Choose
+        one of the kernel functions in ``choclo.prism``.
+    constant_factor : float
+        Constant factor that will be used to multiply each element of the
+        ``fields`` array.
+
+    Notes
+    -----
+    The constant factor is applied here to each element of fields because
+    it's more efficient than doing it afterwards: it would require to
+    index the elements that corresponds to each component.
+    """
+    n_receivers = receivers.shape[0]
+    n_nodes = nodes.shape[0]
+    n_cells = cell_nodes.shape[0]
+    # Evaluate kernel function on each node, for each receiver location
+    for i in prange(n_receivers):
+        # Allocate vector for kernels evaluated on mesh nodes
+        kernels = np.empty(n_nodes)
+        for j in range(n_nodes):
+            kernels[j] = _evaluate_kernel(
+                receivers[i, 0],
+                receivers[i, 1],
+                receivers[i, 2],
+                nodes[j, 0],
+                nodes[j, 1],
+                nodes[j, 2],
+                kernel_func,
+            )
+        # Compute fields from the kernel values
+        for k in range(n_cells):
+            fields[i] += (
+                constant_factor
+                * densities[k]
+                * _kernels_in_nodes_to_cell(
+                    kernels,
+                    cell_nodes[k, 0],
+                    cell_nodes[k, 1],
+                    cell_nodes[k, 2],
+                    cell_nodes[k, 3],
+                    cell_nodes[k, 4],
+                    cell_nodes[k, 5],
+                    cell_nodes[k, 6],
+                    cell_nodes[k, 7],
+                )
+            )
+
+
+def _sensitivity_gravity(
+    receivers,
+    nodes,
+    sensitivity_matrix,
+    cell_nodes,
+    kernel_func,
+    constant_factor,
+):
+    """
+    Fill the sensitivity matrix
+
+    This function should be used with a `numba.jit` decorator, for example:
+
+    ..code::
+
+        from numba import jit
+
+        jit_sensitivity = jit(nopython=True, parallel=True)(_sensitivity_gravity)
+        jit_sensitivity(
+            receivers, nodes, densities, fields, cell_nodes, kernel_func, const_factor
+        )
```

Comment:
I would just remove these lines. They had errors and they are hard to detect. Besides, they are just reproducing the function signature, so they aren't too informative.
```suggestion
```